npm/src/upload/form/update-and-display-success-message.ts

```diff
@@ -47,6 +47,6 @@ export const displaySelectionSuccessMessage = (
   )
 
   successMessageContainer?.removeAttribute("hidden")
-  successMessage?.removeAttribute("hidden")
+  successMessage?.classList.remove("govuk-visually-hidden")
```

Comment:
Judgment upload page still uses the hidden attribute. I think you can just add `govuk-visually-hidden` to the judgment upload view and remove "hidden"